ntoskrnl/se/accesschk.c

```diff
@@ -486,7 +424,8 @@ SepAccessCheck(
     BOOLEAN Defaulted;
     NTSTATUS Status;
     PACCESS_TOKEN Token = NULL;
-    PACCESS_CHECK_RIGHTS AccessCheckRights = NULL;
+    ACCESS_MASK AccessRightsArray[ACCESS_RIGHTS_MAX_ARRAY];
```

Comment:
If you just initialize your data here, you don't need to pass the whole 'initialize' thingy to the function.